health-checker/src/main/java/io/stargate/health/Server.java

```diff
@@ -75,6 +79,7 @@ public void initialize(final Bootstrap<ApplicationConfiguration> bootstrap) {
     super.initialize(bootstrap);
     bootstrap.setConfigurationSourceProvider(new ResourceConfigurationSourceProvider());
     bootstrap.setMetricRegistry(metrics.getRegistry("health-checker"));
+    bootstrap.setHealthCheckRegistry(healthCheckRegistry);
```

Comment:
can we add an integration test that validates if health-check was exposed on the dropwizard `/helalth` endpoint?

Reply:
👍  done